bluesky/traffic/route.py

```diff
@@ -48,7 +51,21 @@ def __init__(self):
         # default: False
         self.flag_landed_runway = False
 
-        return
+        self.iac = self.wpdirfrom = self.wpdistto = self.wpialt = \
+            self.wptoalt = self.wpxtoalt = None

```

Comment:
Code analyzer complains that these are assigned in the methods below without being defined in init.